tests/test_ecephys/test_tools_spikeinterface.py

```diff
@@ -1009,6 +1028,173 @@ def test_write_units_table_in_processing_module(self):
         self.assertEqual(units_table.name, units_table_name)
         self.assertEqual(units_table.description, unit_table_description)
 
+    def test_write_subset_units(self):
+        """ """
+        subset_unit_ids = self.base_sorting.unit_ids[::2]
+        add_units_table(
+            sorting=self.base_sorting,
+            nwbfile=self.nwbfile,
+            unit_ids=subset_unit_ids,
+        )
+
+        self.assertEqual(len(self.nwbfile.units), len(subset_unit_ids))
+        self.assertTrue(all(str(unit_id) in self.nwbfile.units["unit_name"][:] for unit_id in subset_unit_ids))
+
+    def test_write_bool_properties(self):
+        """ """
+        bool_property = np.array([False] * len(self.base_sorting.unit_ids))
+        bool_property[::2] = True
+        self.base_sorting.set_property("test_bool", bool_property)
+        add_units_table(
+            sorting=self.base_sorting,
+            nwbfile=self.nwbfile,
+        )
+        self.assertIn("test_bool", self.nwbfile.units.colnames)
+        assert all(tb in ["False", "True"] for tb in self.nwbfile.units["test_bool"][:])
+
+
+@unittest.skipIf(
```

Comment:
Should we consider dropping python 3.7? I have the impression that is causing problems somewhere else and it will have its eol in six months. Seems like pointless to support this already to me.